src/types/user.types.ts

```diff
@@ -165,3 +167,21 @@ export const emptyEmployment: Employment = {
   endDate: '',
   fullTimeEquivalentPercentage: '',
 };
+
+export interface NviVerification {
```

Comment:
kan ikke se at denne brukes en annen plass enn her
```suggestion
interface NviVerification {
```